src/compiler/interfaces.ts

```diff
@@ -160,4 +160,5 @@ export interface Var {
 	hoistable?: boolean;
 	subscribable?: boolean;
 	is_reactive_dependency?: boolean;
+	argument?: boolean;
```

Comment:
I'd name it `aliased` and work our way into testing for other types of aliasing, as they also break hoisting:

```js
function foo() {}
const foo_alias = foo;
foo_alias.nasty_but_legal = 123;
```

Reply:
Added a test for that